android/settings.gradle

```diff
@@ -22,3 +22,4 @@ include ':walletlib'
 include ':clientlib'
 include ':fakewallet'
 include ':fakedapp'
+include ':clientlib-ktx'
```

Comment:
Nitpick++ 😁 : you could put it just under `include ':clientlib'`

Reply:
No, it's good to keep this organized!